docs/version0.18.md

```diff
@@ -47,6 +48,12 @@ You can set the `-Xshareclasses:noTimestampChecks` option to turn off timestamp
 
 On z/OS 64-bit, [`-Xmso`](xmso.md) has a 1 MB minimum value, to match the minimum stack space provided by the operating system. If you set a value smaller than 1 MB, the value is ignored.
 
+### Add new Xdump agent exit and support for hotspot option ExitOnOutOfMemoryError
+
+Xdump agent "exit" will shutdown the JVM at the specified event. The "exit" agent is at priority level 0 and "tool" agent has been moved to priority level 1 to aid in mimicking the behavior of hotspot options.
+
+OpenJ9 now supports the hotspot option `-XX:[+-]ExitOnOutOfMemoryError`. You can set this option to have the JVM shutdown when a java.lang.OutOfMemory error is thrown by the JVM or in Java code. See the [-XX:[+-]ExitOnOutOfMemoryError](xxexitonoutofmemory.md) option.
+
```

Comment:
Does this need to (briefly) mention the relationship between the 'exit' agent and the ExitOnOutOfMemoryError option? When do I use one or other? (Detail might be better in one or both topics.)